### DISCUSSION TITLE
[Prettier Plugin] Use exact versions in installation instructions

### DISCUSSION THREAD

**Discussion by jaronheard** [tailwindlabs/tailwindcss#9427] [Ideas]
[Prettier's installation instructions](https://prettier.io/docs/en/install.html#summary) tell you to use an exact version `npm install --save-dev --save-exact prettier`.

> Install an exact version of Prettier locally in your project. This makes sure that everyone in the project gets the exact same version of Prettier. Even a patch release of Prettier can result in slightly different formatting, so you wouldn’t want different team members using different versions and formatting each other’s changes back and forth.

It seems that both prettier and the prettier-plugin-tailwindcss should be installed using an exact version for this reason.

Currently the [instructions for the plugin](https://github.com/tailwindlabs/prettier-plugin-tailwindcss#installation) do not specify an exact version:
`npm install -D prettier prettier-plugin-tailwindcss`

**Comment by ADTC** [comment]
_Why_ does Prettier instruct you to install the exact version?

Edit: Never mind, didn't see the blockquote. Me going blind!
